runtime/runtime/src/lib.rs

```diff
@@ -1070,6 +1070,49 @@ impl Runtime {
         Ok(())
     }
 
+    // In test runs reads and writes here used 442 TGas. Added 10% to account for possible bigger
+    // state
+    #[cfg(feature = "protocol_feature_fix_storage_usage")]
+    const GAS_USED_FOR_STORAGE_USAGE_DELTA_MIGRATION: Gas = 490_000_000_000_000;
+
+    pub fn apply_migrations(
+        &self,
+        state_update: &mut TrieUpdate,
+        apply_state: &ApplyState,
+    ) -> Result<Gas, StorageError> {
+        #[cfg(feature = "protocol_feature_fix_storage_usage")]
+        let mut gas_used: Gas = 0;
+        #[cfg(not(feature = "protocol_feature_fix_storage_usage"))]
+        let gas_used: Gas = 0;
+        #[cfg(feature = "protocol_feature_fix_storage_usage")]
+        if ProtocolFeature::FixStorageUsage.protocol_version()
+            == apply_state.current_protocol_version
```

Comment:
What do you think about this approach?
```
let gas_used: Gas = 0;
#[cfg(feature = "protocol_feature_fix_storage_usage")]
let gas_used = if ProtocolFeature::FixStorageUsage.protocol_version()
            == apply_state.current_protocol_version {
   ...
   Runtime::GAS_USED_FOR_STORAGE_USAGE_DELTA_MIGRATION;
}
```

Reply:
It would work while there is single migration, but what if there's more?

Reply:
Got it, thank you!